packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -630,11 +630,34 @@ class WebKitWebViewController extends PlatformWebViewController {
   }
 
   Future<void> _injectConsoleOverride() {
+    // Using the replacer parameter of JSON.stringify() to solve the error
+    // TypeError: JSON.stringify cannot serialize cyclic structures.
+    // See https://github.com/flutter/flutter/issues/144535.
+    //
+    // Considering this is just looking at the logs printed via console.log,
+    // the cyclic object is not important, so remove it.
     const WKUserScript overrideScript = WKUserScript(
       '''
+function removeCyclicObject() {
+  const levelObjects = [];
```

Comment:
I'm not sure what this naming means; what is a "level object"?

Reply:
`levelObjects` is used to record objects whose properties are currently being traversed. (Maybe I should rename it to `stack` or something? )

As shown in the figure below, the properties of the outermost object (blue box) are currently being traversed, and obj1 (green box) is also an object, so the properties of obj1 will be traversed next and pushed in levelObjects.

<img width="974" alt="image" src="https://github.com/flutter/packages/assets/19367531/7d292e4b-f8c3-401f-9c08-70bce40d795d">

When traversing all the properties of obj1 (green box) and starting to traverse obj2, currentParentObj will point to the outermost object (blue box) and the last element obj1 (green box) of the current `levelObjects` is not equal, then obj1 will be popped up, and obj2 will be pushed to levelObjects.

<img width="1040" alt="image" src="https://github.com/flutter/packages/assets/19367531/16d9a533-1911-490a-aa1e-e9ca56367539">